build.gradle

```diff
@@ -4,19 +4,24 @@ plugins {
 
 group 'org.testcontainers.cloud.turbomode.sample'
 version '1.0-SNAPSHOT'
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
 
 repositories {
     mavenCentral()
 }
 
 dependencies {
     testImplementation 'org.junit.jupiter:junit-jupiter:5.9.0'
-    testImplementation platform('org.testcontainers:testcontainers-bom:1.17.4')
+    testImplementation platform('org.testcontainers:testcontainers-bom:1.17.5')
     testImplementation "org.testcontainers:testcontainers"
     testImplementation "org.testcontainers:junit-jupiter"
     testImplementation 'ch.qos.logback:logback-classic:1.3.3'
     testImplementation 'org.assertj:assertj-core:3.23.1'
```

Comment:
Looks like it is not used.

Reply:
It was used in `TestcontainersCloudFirstTest` but I changed to use assertions from jupiter as well, this way we can remove the dependency